packages/protobuf/src/messages-schema.ts

```diff
@@ -1,15 +1,5 @@
+import { DeviceModelInternal } from '@trezor/device-utils';
 import { CloneType, Static, Type } from '@trezor/schema-utils';
-
-export enum DeviceModelInternal {
```

Comment:
This is an auto-generated file, to do this the scripts would need updating

Reply:
yep, I missed that

Reply:
and I can't say I like it very much making this types generation script even more complex :(